src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -181,22 +188,25 @@ object SparkCassandraStress {
       }
 
     val wallClockStartTime = System.nanoTime()
-    val time = test.runTrials(sc)
+    val timesAndOps: Seq[(Long,Long)]= test.runTrials(sc)
+    val time = for (x <- timesAndOps) yield {x._1}
+    val totalCompletedOps = for (x <- timesAndOps) yield {x._2}
 
     val wallClockStopTime = System.nanoTime() 
     val wallClockTimeDiff = wallClockStopTime - wallClockStartTime
     val wallClockTimeSeconds = (wallClockTimeDiff / 1000000000.0) 
-    val timeSeconds = time.map{ _ / 1000000000.0}
-    val opsPerSecond = timeSeconds.map{ config.totalOps/_}
+    val timeSeconds = time.map{ x => math.round( x / 1000000000.0 ) }
+    val opsPerSecond = for (i <- 0 to timeSeconds.size-1) yield {math.round(totalCompletedOps(i)/timeSeconds(i))}
```

Comment:
Rounding here saves us from having to do it later, and gives us a more tractable opsPerSecond readout that a user can easily verify by dividing the number of inserts actually made by the runtime.